examples/resources/powerscale_synciq_certificate/resource.tf

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright (c) 2024 Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the Mozilla Public License Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://mozilla.org/MPL/2.0/
+
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+
+# Available actions: Create, Read, Update, Delete and Import.
+
+# Step 1: Upload a certificate to PowerScale.
+# This can be done using the file provisioner.
+resource "terraform_data" "cert" {
+  provisioner "file" {
+    source      = "/root/certs/pscale.crt" # certificate that user has been given
```

Comment:
Existing Comment: Certificate that user has been given
Proposed Comment: Certificate that user has given